### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by nyior**
- Link to our community Slack
-  Basic contributing guide

### CODE REVIEW

**Comment by kickster97** [comment]
Great! 
Should we post an invite link instead of just the slack-community link? Now people who are not already a part of it do not get access.

**Comment by nyior** [comment]
> Great! Should we post an invite link instead of just the slack-community link? Now people who are not already a part of it do not get access.

Right! I'll update that. Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+You are interested in contributing to LavinMQ, but you are not sure how to? 
+Right now, you can contribute to LavinMQ in one of two ways:
+
+1. [Code contributions](#code-contributions)
+2. [Non-code contributions](#non-code-contributions)
+
+## Code contributions
```
Comment (carlhoerberg) [inline CONTRIBUTING.md:7]: (standard) markdown requires a newline between a header and the paragraph.
```suggestion
## Code contributions

```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+1. Create and push an annotated tag (`git tag -a v$(shards version)`), put the changelog of the version in the tagging message
+
+## Non-Code contributions
+Your schedule won't allow you make code contributions? Still fine, you can:
+
+### [Report an issue](https://github.com/cloudamqp/lavinmq/issues/new)
+- This could be an easily reproducible bug or even a feature request.
+- If you spot an unexpected behaviour but you are not yet sure what the underlying bug is, the best place to post is [LavinMQ's community Slack](https://join.slack.com/t/lavinmq/shared_invite/zt-1v28sxova-wOyhOvDEKYVQMQpLePNUrg). This would allow us to interactively figure out what is going on. 
+
+### [Give us some feedback](https://github.com/cloudamqp/lavinmq/issues/new)
```
Comment (carlhoerberg) [inline CONTRIBUTING.md:34]: link to https://github.com/cloudamqp/lavinmq/discussions instead?

File: README.md [hunk 3]
```diff
@@ -206,6 +188,11 @@ When `Client#read_loop` receives a Basic.Consume frame it will create a `Consume
 the queue's list of consumers. Each consumer has a `deliver_loop` fiber that will be notified
 by an internal `Channel` when new messages are available in the queue.
 
+## Getting help
+For questions or suggestions:
+- We are on [Slack](https://join.slack.com/t/lavinmq/shared_invite/zt-1v28sxova-wOyhOvDEKYVQMQpLePNUrg). 
+- You can also use the [lavinmq tag](https://stackoverflow.com/questions/tagged/lavinmq) on Stackoverflow
+
```
Comment (carlhoerberg) [inline README.md:195]:
```suggestion
- If you use LavinMQ via [CloudAMQP](https://www.cloudamqp.com) then reach out to [support@cloudamqp.com]
```

File: README.md [hunk 4]
```diff
@@ -206,6 +188,11 @@ When `Client#read_loop` receives a Basic.Consume frame it will create a `Consume
 the queue's list of consumers. Each consumer has a `deliver_loop` fiber that will be notified
 by an internal `Channel` when new messages are available in the queue.
 
+## Getting help
```
Comment (carlhoerberg) [inline README.md:191]:
```suggestion
## Getting help

```

**Review by carlhoerberg — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (nyior) [reply CONTRIBUTING.md:34]: > link to https://github.com/cloudamqp/lavinmq/discussions instead?

I agree

**Pull request merged**